SSH2.xs

```diff
@@ -2048,7 +2048,7 @@ CODE:
 SSH2_CHANNEL*
 net_ls_accept(SSH2_LISTENER* ls)
 PREINIT:
-    SSH2* ss;
+    SSH2* ss = 0;
```

Comment:
I think this solution is wrong, you have to use the value from the `ss` slot of the structure `SSH2_LISTENER` in `ls`. Probably adding the following line in the `CODE` section:

     ss = ls->ss;

Reply:
well, with this change it's better than before... :-) However, I'll improve this.